templates/patterns/list_with_illustration/list_with_illustration.ui_patterns.yml

```diff
@@ -0,0 +1,52 @@
+list_with_illustration:
+  label: "List with illustration"
+  description: "A list of items with associated illustrations."
+  variants:
+    vertical_images_large:
+      label: "Vertical with images (large)"
+      description: "Vertical list with large images."
+    vertical_images_square:
+      label: "Vertical with images (square)"
+      description: "Vertical list with square images."
+    vertical_icons:
+      label: "Vertical with icons"
+      description: "Vertical list with icons."
+    horizontal_images_large:
+      label: "Horizontal with images (large)"
+      description: "Horizontal list with large images."
+    horizontal_images_square:
+      label: "Horizontal with images (square)"
+      description: "Horizontal list with square images."
+    horizontal_icons:
+      label: "Horizontal with icons"
+      description: "Horizontal list with icons."
+  fields:
+    column:
+      type: "numeric"
+      label: "Column number"
+      description: "Number of columns (only applies to horizontal variants). Maximum of 4 columns."
+      preview: 4
```

Comment:
Can we either have 4 items or only 2 columns for the preview? Looks odd like this in horizontal variants xD